### PR TITLE
test create new QuickText

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

File: cypress/integration/quicktest/create-quicktext.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const endPoint = require('../../fixtures/endpoint/endpoint.json')
+const {pageTransporter} = require('../../src/salesforce/transporter')
+const quickText = require('../../src/salesforce/quickText/quickText')
+const newQuickText = require('../../src/salesforce/quickText/new-quicktext')
+const detailQuickText = require('../../src/salesforce/quickText/detail-quicktext')
+
+describe('test for feature Quick Text', () => {
+    const name = 'Quick Text Name'
```
Comment (jpgonzales27) [inline cypress/integration/quicktest/create-quicktext.js]: Maybe these variables should be moved to a JSON file

File: cypress/support/utils.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Cypress.Commands.add('readFile', (path) => {
```
Comment (JRojas999) [inline cypress/support/utils.js]: This is not necessary if we're using require to import files.

File: cypress.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
     "chromeWebSecurit": false,
     "watchForFileChanges": false,
-    "baseUrl" : "",
+    "baseUrl" : "https://jala3-dev-ed.my.salesforce.com/",
     "env" : {
-        "username" : "",
-        "password" : ""
+        "username" : "edsonarios@freeorg01.com",
```
Comment (jpgonzales27) [inline cypress.json:6]: Should be removed your credentials

Comment (edsonarios) [reply cypress.json:6]: The .env file must be implemented, but it will be implemented in the next iteration

**Review by JRojas999 — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Pull request merged**